src/main/java/domain/RacingGame.java

```diff
@@ -35,15 +33,22 @@ private void moveCar(final Car car) {
     }
 
     public List<Car> getCars() {
-        return List.copyOf(cars);
+        List<Car> copiedCars = cars.stream()
+                .map(Car::new)
+                .collect(Collectors.toList());
+        return List.copyOf(copiedCars);
     }
 
     public List<Car> getWinners() {
+        List<Car> copiedCars = cars.stream()
+                .map(Car::new)
+                .collect(Collectors.toList());
+
         Car furthestCar = getFurthestCar();
 
-        return cars.stream()
+        return copiedCars.stream()
                 .filter(car -> car.getPosition() == furthestCar.getPosition())
-                .collect(Collectors.toList());
+                .collect(Collectors.toUnmodifiableList());
```

Comment:
반환할 우승자 객체만 복사해도 되겠네요!

```suggestion
        Car furthestCar = getFurthestCar();
        return cars.stream()
                .filter(car -> car.getPosition() == furthestCar.getPosition())
                .map(Car::new)
                .collect(Collectors.toUnmodifiableList());
```

Reply:
좋은 지적감사합니다!
애초에 반환할 것이 우승 자동차라면, 모든 자동차를 복사한 뒤 필터를 해서 반환하는게 아닌,
필터를 한 뒤 복사해서 생성하는게 낫겠네요 감사합니다!